### ISSUE TITLE
PHP Fatal Error on new webhook

### ISSUE THREAD

**Issue by daltonrooney** [craftcms/webhooks#34]
Currently testing Webhooks 2.3.1 with Craft CMS 3.5 beta 3
Clicking on new webhook generates a fatal error: 

```
yii\base\ErrorException: Class craft\webhooks\filters\NewElementFilter contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (craft\base\ComponentInterface::isSelectable) in /home/user/webapps/installation/releases/1593279945256/craft/vendor/craftcms/webhooks/src/filters/NewElementFilter.php:18
Stack trace:
#0 [internal function]: yii\base\ErrorHandler->handleFatalError()
#1 {main}
```

**Comment by brandonkelly** [comment]
Thanks for reporting that! Just released Webhooks 2.3.2 with a fix for that.

**Comment by daltonrooney** [comment]
Thanks, I appreciate that on a weekend, too!
